modules/nw-ingress-reencrypt-route-custom-cert.adoc

```diff
@@ -0,0 +1,54 @@
+// This is included in the following assemblies:
```

Comment:
Hmmm. After making my comments, I'm considering what this would be like if you just added it to the `Creating a route through an Ingress object` section. I'm thinking that this doesn't necessarily warrant a whole section to itself.

If you look at the section `Creating a route through an Ingress object`, you'll see we talk about `route.openshift.io/termination: "reencrypt"` with a very detailed footnote (1). We just need a small footnote (2) with a combination of 1. What it does 2. Any Prerequisites. 3. The fact that it is optional (just like footnote 1)

Then, the "Example Output" can be merged to add `tls.destinationCACertificate`. Let me know if this makes sense. I will let you be the judge. I think it would fit, but you could argue it might start to get a little crowded.

Reply:
Funny, after we had our Slack convo, I was also thinking all this info could just go in the `Creating a route through an Ingress object` topic too but then as I started adding and considering prerequisites too I did think it may be too crowded. Like the footnote 1 does have an a) under it and technically a footnote 2 could have the same. Idk maybe it wouldn't really be so bad. I understand what you mean, its an optional thing so it doesn't really need to emphasis of its own subtopic. Let me play around and just seee.